src/utilities/new_versions.jl

```diff
@@ -54,7 +55,7 @@ function pr_info(
     pr_title_prefix::String,
 )
     new_pr_title = m"""
-    $(pr_title_prefix)CompatHelper: add new compat entry for $(name) at version
+    $(pr_title_prefix)CompatHelper: add new compat entry for $(name) in [$(section)] at version
```

Comment:
I'd rather not change the PR title for regular deps. Can we keep the old PR title for regular deps, and only include the "in section" text for weak deps?

Reply:
Done!